web/server/entrypoint_test.go

```diff
@@ -265,10 +288,6 @@ func TestMachineState(t *testing.T) {
 	go func() {
 		defer wg.Done()
 
-		// Create a temporary config file with a single
-		tempConfigFile, err := os.CreateTemp(t.TempDir(), "temp_config.json")
```

Comment:
replaced this pattern all over this file because windows is a lot pickier about being able to remove currently open files